### DISCUSSION TITLE
"reverse engeneered ChatGPT"

### DISCUSSION THREAD

**Discussion by MentalGear** [acheong08/ChatGPT#438] [General]
Project seems quite interesting, but to be credible, you might not want to start the readme with the claim that your "reverse engeneered ChatGPT". The API, yes, but certainly not the model. Cheers

**Comment by acheong08** [comment]
That is true. I will change it. I haven't properly updated the readme in a while and forgot about it

**Reply by acheong08** [reply]
2019073f767bcfafadb70721f6ec995060638ab8 updated

**Reply by acheong08** [reply]
(This should be in GitHub issues as it is a pretty important flaw. I don't want to get sued lol)

**Reply by MentalGear** [reply]
Thanks for your work !
